README.md

```diff
@@ -34,22 +34,27 @@ The application requires Key Vault and Cosmos DB to be setup per the Helium [rea
 - Fork this repo and clone to your local machine
   - cd to the base directory of the repo
   - All instructions assume starting from the root of the repo
-
-## CI-CD
-
-
-This repo uses [GitHub Actions](/.github/workflows/dockerCI.yml) for Continuous Integration.
-
-- CI supports pushing to Azure Container Registry or DockerHub
-- The action is setup to execute on a PR or commit to ```master```
-  - The action does not run on commits to branches other than ```master```
-- The action always publishes an image with the ```:beta``` tag
-- If you tag the repo with a version i.e. ```v1.0.8``` the action will also
-  - Tag the image with ```:1.0.8```
-  - Tag the image with ```:stable```
-  - Note that the ```v``` is case sensitive (lower case)
-
-CD is supported via webhooks in Azure App Services connected to the ACR or DockerHub repository.
+  
+- Setup using [Codespace](https://visualstudio.microsoft.com/services/visual-studio-codespaces/)
+  - Fork this repo
+  - Create a new CodeSpace pointing to the forked repo
+  - NOTE: While the Codespace is getting prepared, this popup shows up which gets resolved once the mvn gets installed
+     ![popup](popup.jpg)
+        
+  - Quick start with F5
```

Comment:
need to run az login first (or copy from your dotfiles)

Reply:
Added this to the readme